gwwc-data-table/ingest_us_tax_deductibility.py

```diff
@@ -0,0 +1,38 @@
+from selenium import webdriver
+import csv
+from difflib import SequenceMatcher
+
+# Charity Navigator only displays organizations that are tax deductible in the US, so we can scrape
+# it to see which organizations on our list are tax deductible in the US
+
+def update_us_tax_deductibility():
+    driver = webdriver.Firefox()
+    result_rows = []
+    with open('../../List of Organisations _ Charities recommended _ donated _ granted to (potentially for GWWC Giving Recommendations page) - Organisations.csv', 'r') as csv_file:
```

Comment:
Can we change the filename to a variable that is passed into the script e.g. call it as `python ingest_us_tax_deductability.py in.csv out.csv` and use the `sys` library to access the passed arguments